examples/simple_fastapi_app/README.md

```diff
@@ -0,0 +1,328 @@
+# Usage 
+
+This directory contains a simple [fastAPI Application](with_psycopg.py) that
+uses pgai Vectorizer to perform semantic search and RAG. The app will ingest
+some wikipedia articles, create vector embeddings for them, and allow you to
+perform semantic search and RAG. 
+
+To run the app, first download the file with the following command:
+
+```
+curl -O https://raw.githubusercontent.com/timescale/pgai/main/examples/simple_fastapi_app/with_psycopg.py
+```
+
+Create a `.env` file with the following:
+
+```
+OPENAI_API_KEY=<your-openai-api-key>
+DB_URL=<your-database-url>
+```
+
+and then you can use the following command:
+
+```
+fastapi dev with_psycopg.py
+```
+
+By going to ` http://0.0.0.0:8000/docs` you can see the API documentation and try out the endpoints provided by the app.
+
+The main endpoints are:
+- `/insert`: Insert a row into the wiki table.
+- `/search`: Search the articles using semantic search.
+- `/rag`: Perform RAG with the LLM.
+
+
+# Code walkthrough
+
+We'll walk you through the main parts of the code below. 
+
+1. **Enable pgai vectorizer on your database**
+    During app startup we run the following Python to install the necessary database object in your PostgreSQL database. All the database objects are installed in the `ai` schema. Note that in production use cases, this can also be done via a database migration.
+    
+    ```python
+    pgai.install(DB_URL)
+    ```
+    
+    We also run the vectorizer worker as part of the FastAPI app lifecycle. 
+    
+    ```python
+    worker = Worker(DB_URL)
+    task = asyncio.create_task(worker.run())
+    ```
+    
+    In this example, we run the Vectorizer worker inside the FastAPI app for simplicity. You can also run the vectorizer worker outside the FastAPI app, in a separate process or separate container. Please see the [vectorizer worker](/docs/vectorizer/worker.md) documentation for more information.
+    
+1. **Create the table and load the test dataset**
+
+    We'll create a table named `wiki` from a few rows of the english-language `wikimedia/wikipedia` dataset.
```

Comment:
I notice we are mixing future and present tense. I think we should use present one always here.
```suggestion
    We create a table named `wiki` from a few rows of the english-language `wikimedia/wikipedia` dataset.
```

This applies to some other parts of this walkthrough